src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -239,7 +276,7 @@ public function returnImage($id, $formatName)
      *
      * @throws ImageProxyInvalidImageFormat
      */
-    protected function getFormat($format)
+    private function getFormat($format)
```

Comment:
It's generally best practice to keep the visibility as low as possible. There was no reason the make this method protected and I am very confident that nobody has extended the FormatManager. How would you like me to proceed?